app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java

```diff
@@ -489,7 +489,7 @@ protected void gatherData() {
             username = usernameEditText.getText().toString();
             password = passwordEditText.getText().toString();
 
-            url = "https://v2.wallabag.org";
+            url = "https://www.wallabag.it";
```

Comment:
Shouldn't it be "https://app.wallabag.it"? I did not test it in the app, though.